helm-charts/charts/flink/templates/jobmanager.yaml

```diff
@@ -19,7 +19,7 @@ spec:
     spec:
       containers:
       - name: jobmanager
-        image: flink:1.11.2-scala_2.12
```

Comment:
hey ben can we change this to flink 1.11.1 ?    one of the packages i need (i think flink-clients) isnt available for 1.11.2

Reply:
I think we need java other then scala